go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -83,6 +83,9 @@ var (
 
 	// path component delimiter
 	delimiter = "/"
+
+	// use a shared transport for all connections
+	defaultS3Transport *http.Transport
```

Comment:
Nit, but s3 is redundant here. We can just call it `defaultTransport` as s3 is the package.

Reply:
got rid of the global variable 😌